Chinese/Mr.X/你们真会玩儿

```diff
@@ -0,0 +1 @@
+从前有个山，山里有个庙，庙里有个小和尚和一个老和尚，小和尚对老和尚说，爸！吃饭啦！
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.